tests/wpt/include.ini

```diff
@@ -5,6 +5,8 @@ skip: true
   skip: false
 [2dcontext]
   skip: false
+  [imagebitmap]
+    skip: true
```

Comment:
Just remove this and r=me, it's redundant with `__dir__.ini` and this mechanism tends to behave weirdly if you pass explicit test paths to `./mach`.